forms-flow-api/src/formsflow_api/services/external/bpm.py

```diff
@@ -116,6 +117,12 @@ def post_deployment(cls, token, payload, tenant_key, files):
         url = f"{cls._get_url_(BPMEndpointType.DEPLOYMENT)}"
         return cls.post_request(url, token, payload, tenant_key, files)
 
+    @classmethod
+    def complete_task(cls, task_id, payload, token):
+        """Complete task."""
+        url = f"{cls._get_url_(BPMEndpointType.COMPLETE_TASK).replace('{task_id}', task_id)}"
```

Comment:
We can use a plain string here instead of f-string.
`f"{cls._get_url_(BPMEndpointType.COMPLETE_TASK).replace('{task_id}', task_id)}"` 
is same as 
`cls._get_url_(BPMEndpointType.COMPLETE_TASK).replace('{task_id}', task_id)`.